lib/index.js

```diff
@@ -355,18 +382,19 @@ Publisher.prototype.publish = function (headers, options) {
 
 /**
  * Sync file in stream with file in the s3 bucket
- * @param {String} prefix
+ * @param {String} prefix prefix to sync a specific directory
+ * @param {Array} whitelist list of expressions that match against files that should not be deleted
```

Comment:
Could you rename the name of the param here too to whitelistedFiles? Pretty please :)